CompuCell3D/core/post_install_tasks/config.py.in

```diff
@@ -0,0 +1,13 @@
+"""Defines installation-specifc information"""
+
+versionMajor = @COMPUCELL3D_MAJOR_VERSION@
+versionMinor = @COMPUCELL3D_MINOR_VERSION@
+versionBuild = @COMPUCELL3D_BUILD_VERSION@
+revisionNumber = "@COMPUCELL3D_REV_NUMBER@"
+
+cc3d_install_prefix_rel = '@cc3d_install_prefix_rel@'
```

Comment:
I wonder if we need those at all but you probably had the reason that I am not seeing now. We can always revisit stuff like those extra paths later . I am just tagging it so that later we remember potential places for code simplification